zerver/webhooks/netlify/view.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any, Dict, Iterable, Optional
+
+from django.http import HttpRequest, HttpResponse
+from django.utils.translation import ugettext as _
+
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_error, json_success
+from zerver.lib.validator import check_dict, check_string
+from zerver.models import UserProfile
+
+@api_key_only_webhook_view('Netlify')
+@has_request_variables
+def api_netlify_webhook(
+        request: HttpRequest, user_profile: UserProfile,
+        payload: Dict[str, Iterable[Dict[str, Any]]]=REQ(argument_type='body')
+) -> HttpResponse:
+
+    message_template = (
+        u'State: {}\n'
+        u'Name: {}\n'
+        u'Branch: {}'
+    )
```

Comment:
Okay, so the general pattern with message templates is that they should be a complete and grammatical sentence or paragraph of some sort.

Also, we should always include the URL of the build, without it, the notifications are fairly useless.

Here's an example of a message template:
`The build [{build_name}]({build_url}) on branch {branch_name} is now {state}.`

You might need to change the structure of the template to suit the specific payload/event type. :)